bin/pgenv

```diff
@@ -438,10 +438,28 @@ pgenv_current_postgresql_version(){
 #
 # If no version is specified, the default configuration
 # file is provided.
+#
+# If the user has defined an environment variable named
+#       PGENV_CONFIGURATION_FILE
+# the function returns such name. This allows the user
+# to use always a well established configuration file.
+# It is possible to avoid the usage of the PGENV_CONFIGURATION_FILE
+# environment variable if the second argument is something not null.
 pgenv_configuration_file_name(){
     local v=$1
+    local skip_env=$2
     local PGENV_DEFAULT_CONFIG_FILE="${PGENV_CONFIG_ROOT}/default.conf"
 
+
+    # if the user has defined an environment variable PGENV_CONFIGURATION_FILE
+    # I need to use that instead of trying to understand which file to use
+    if [ ! -z "$PGENV_CONFIGURATION_FILE" -a -z "$skip_env" ]; then
+	pgenv_debug "Configuration file overwritten by environment variable PGENV_CONFIGURATION_FILE = $PGENV_CONFIGURATION_FILE"
```

Comment:
Does this actually overwrite the file, or do you mean to say here that the setting of `PGENV_CONFIGURATION_FILE` overrides the calculated configuration file?